dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  hpp-demo:
+    init: true
+    image: hpp-demo
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+    environment:
+      - DISPLAY=${DISPLAY}
+      - QT_X11_NO_MITSHM="1"
+    network_mode: "host"
+    privileged: true
+    stop_signal: SIGINT
+    security_opt:
+      - seccomp:unconfined
```

Comment:
why do you need privileged and seccomp  here ?

Reply:
This comes from a copy-paste. I am not sure I need it here.

Reply:
After double checking what those option do, I remember why they were in the original file. Indeed it has nothing to do here.
Removed.